test.ts

```diff
@@ -16,18 +17,17 @@ describe('update', () => {
       expect(update([1], {$push: [7]})).toEqual([1, 7]);
     });
     it('does not mutate the original object', () => {
-      const obj = [1];
-      update(obj, {$push: [7]});
-      expect(obj).toEqual([1]);
```

Comment:
Removed this `expect(obj).toEqual(xxx)` paradigm from the entire test file, as it's not actually testing what we want to test (i.e., whether the object was mutated).